app/src/components/inputs/inputUserSearch/inputUserSearch.stories.jsx

```diff
@@ -29,28 +29,491 @@ import { InputUserSearch } from './inputUserSearch';
-    return (
-      <InputUserSearch
-        isAdmin
-        projectId="superadmin_personal"
-        onChange={action('Select user')}
-      />
-    );
-  });
-
+  .addDecorator(withReadme(README))
+  .add('default state', () => <InputUserSearch />)
+  .add(
+    'Admin,projectId="superadmin_personal",with actions (type "test" in input, close DevTools, you get MOCK DATA)',
+    () => {
+      const mock = new MockAdapter(axios);
+      const API_REQUEST =
+        '/api/v1/user/search/?page.page=1&page.size=10&page.sort=login,ASC&term=test';
+      mock.onGet(API_REQUEST).reply(200, {
+        content: [
+          {
+            userId: '.elogin.new',
+            email: 'elogin@gmaile.com',
+            full_name: 'test',
+            account_type: 'INTERNAL',
+            userRole: 'ADMINISTRATOR',
```

Comment:
Use roles, accountTypes constants instead of strings. And please fix all below cases